shared/localization/format.js

```diff
@@ -160,6 +159,21 @@ function _preformatValues(messageFormatter, values, lhlMessage) {
   return formattedValues;
 }
 
+/**
```

Comment:
just found there is a `ignoreTag` option we could use

Reply:
Does `ignoreTag` work? I'm fine with this workaround it just doesn't seem ideal.